src/debian/snmp/virt-df.sh

```diff
@@ -1,49 +1,76 @@
 #!/bin/bash
+
```

Comment:
Please add a small description of what this script is doing.
Also, it needs copyright and license.

Reply:
ok